IDE-Proposals/IDE-Proposal-RecruitmentSites.md

```diff
@@ -0,0 +1,24 @@
+
+# Title
+A one-stop diversity focused reference site for hiring managers.
+
+## Summary 
+We are all trying to get more Blacks and underrepresented minorities into our workplaces. We need hiring managers to start advertising in different places. The current places are clearly not bringing in the diverse candidate pool we need. Too often hiring managers say they have advertised broadly, that is not what we need, they need to advertise to a niche.
+
+## Team 
+Finding the jobs sites and building the database is work that a minority student could be paid to do. 
+Building the associated website with advanced search functionality could be work performed by a minority computer science student.
+In both cases preferably from a HBCU.
+The work has been started by volunteers: https://ranpalgill.github.io/HUM/
+I would like to manage the team and activities.
+
+## Plan
+This proposal is to create a searchable list of jobs sites that have a committment to minorities. Where we come to look for open positions knowing that employers have specifically sought out to post their opening on a jobs site that is committed to serving minorities. For job seekers to know they are in a safe space applying to committed employers.
+First order is to build the list of sites. Second order is to build a website that allows advanced search of the data. Finally this tool needs to get into the hands of recruiters and hiring managers. 
+Stretch goals could be for jobs seekers and hiring managers to be able to review and comment on the jobs sites for their effectiveness.
+
+## Impact
+It deals with a bottle neck in the process of getting underrepresented minorities into the astro and STEM fields/workplaces in general, if the work of this proposal can help to improve diverse hiring, attract  minority talent to the field then it ultimately helps with diversity of the AstroPy community and project.
+
+## Budget
+Paying students to search for the data to build the database of sites and also to create a searchable website. Purchasing a URL and hosting (optional). This is the section of the proposal that I need help to flesh out.
```

Comment:
There are at least two aspects requiring different resources - I can comment on Coding website improvements. This is endless, I am an inexperienced JavaScripter so putting up the prototype site backed by the google sheet took me a weekend. Call it 10 hours perhaps more.  If we look for someone experienced (and from a minority!) we should at least consider 80 hours for improvements like better data entry on back end, cleaner design, perhaps not a google sheet. Or go as for someone to do a requirements and analysis and quote .

Reply:
I can comment on finding resources.  The initial search and populating the google sheet was also a weekend (some resources were given to me but still required research).  More time should uncover more resources. So this could also be another 80 hours - a person familiar with researching should be able to significantly increase the number of rseources in the database and I have some starting points that i just haven't had the time to look into.

Both the website improvements and searching for resources are on going so one could put as many hours as there is money available (I listed some stretch goals for the website). It feels inappropriate to look for units less than 40 hours.  

For the website development part we could ask for some quotes based on problem description.